src/pages/components/navbar/Navbar.jsx

```diff
@@ -1,11 +1,14 @@
 import React from 'react'
 import {Link } from "react-router-dom"
+
 import '../navbar/navbar.css'
-const Navbar = () => {
+const Navbar = (mode) => {
+  console.log(mode);
```

Comment:
Avoid pushing console logs to the repo

Reply:
noted, will make sure i run prettier. need the logs for future work , so going to keep them till version 1.0
thanks